src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/CosmosFhirDataStore.cs

```diff
@@ -543,6 +555,20 @@ private void UpdateSortIndex(FhirCosmosResourceWrapper cosmosWrapper)
             }
         }
 
+        private static string RemoveVersionIdAndLastUpdatedFromMeta(string rawResource)
+        {
+            // Removing versionId and lastUpdated from meta
+            // We are ignoring meta.versionId and meta.lastUpdated
+            // if other meta elements like Source/Tag/Profile/Security are updated then we will have to insert a new veriosn for a resource
+
+            JObject jObject = JObject.Parse(rawResource);
```

Comment:
Is it possible to remove the version id and last updated time without parsing the resource? If I remember correctly this isn't a very performant operation. We know the values of version and last updated from the top level properties, so it might be possible to do a find/replace to get rid of them.